docusaurus.config.js

```diff
@@ -84,8 +84,8 @@ module.exports = {
         },
         {
           type: "doc",
-          docId: "apis-clients/working-with-apis-clients",
-          label: "APIs & Clients",
+          docId: "apis-tools/working-with-apis-tools",
```

Comment:
I'm uncertain how well docusaurus will handle this change, given it's a single URL that gets used across all versions. I see that you added a bunch of redirects to try to correct things for non-next versions, but I don't think docusaurus will let you get far enough that you'll actually hit the redirect. Because of that, I'm unsure if this approach of changing the top nav URL now, when only `next` has the updated URL, will give better results than waiting to change the top URL until `current` also has it. 

Throwing a wrench into this is that we don't see the effect of redirects when running `npm start`. The good news is that we have [instructions for running locally with Docker](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#testing-redirect-rules), which includes the redirect functionality.

I'm planning to run this branch with docker to play around a little. @christinaausley  Are you interested in me throwing an hour on our calendars to do it together?

Reply:
Sounds great @pepopowitz 😄 Feel free to pop some time on my calendar and we can discuss.

Reply:
If you want to see an example of how this might look when live, go to version 0.25, then click from Guides to APIs & Clients. It jumps to another version because that tab didn't exist in that previous version. Similar but different since this section did not exist initially - https://github.com/camunda/camunda-platform-docs/pull/391